pkgs/development/python-modules/pytmx/default.nix

```diff
@@ -10,16 +10,18 @@
 
 buildPythonPackage rec {
   pname = "pytmx";
-  version = "3.31";
+  version = "3.32";
   format = "setuptools";
```

Comment:
```suggestion
  pyproject = true;
```
and add `setuptools` to `nativeBuildInputs`

Reply:
Using `setuptools-scm` instead of `setuptools` worked. Thanks.